tsl/test/expected/cagg_drop_chunks.out

```diff
@@ -145,184 +145,3 @@ NOTICE:  drop cascades to 32 other objects
 DROP TABLE records;
 DROP TABLE clients;
 \set VERBOSITY default
--- Test that a continuous aggregate is refreshed correctly inside given chunk,
--- so it can be used together with user retention.
--- Shows that it is not affected by issue #2592.
-CREATE OR REPLACE FUNCTION test_int_now() returns INT LANGUAGE SQL STABLE as 
-    $$ SELECT 125 $$;
-CREATE TABLE conditions(time_int INT NOT NULL, device INT, value FLOAT);
-SELECT create_hypertable('conditions', 'time_int', chunk_time_interval => 10);
-    create_hypertable    
--------------------------
- (3,public,conditions,t)
-(1 row)
-
-INSERT INTO conditions
-SELECT time_val, time_val % 4, 3.14 FROM generate_series(0,100,1) AS time_val;
-SELECT set_integer_now_func('conditions', 'test_int_now');
- set_integer_now_func 
-----------------------
- 
-(1 row)
-
-CREATE MATERIALIZED VIEW conditions_7
-    WITH (timescaledb.continuous, timescaledb.materialized_only = TRUE)
-    AS
-        SELECT time_bucket(7, time_int) as bucket,
-            SUM(value), COUNT(value)
-        FROM conditions GROUP BY bucket WITH DATA;
-NOTICE:  refreshing continuous aggregate "conditions_7"
-HINT:  Use WITH NO DATA if you do not want to refresh the continuous aggregate on creation.
-CREATE VIEW see_cagg AS SELECT * FROM conditions_7 WHERE bucket <= 70 ORDER BY bucket;
-SELECT * FROM see_cagg;
- bucket |  sum  | count 
---------+-------+-------
-      0 | 21.98 |     7
-      7 | 21.98 |     7
-     14 | 21.98 |     7
-     21 | 21.98 |     7
-     28 | 21.98 |     7
-     35 | 21.98 |     7
-     42 | 21.98 |     7
-     49 | 21.98 |     7
-     56 | 21.98 |     7
-     63 | 21.98 |     7
-     70 | 21.98 |     7
-(11 rows)
-
--- This is the simplest case, when the updated bucket is inside a chunk, so it is expected
--- that the update is refreshed.
-UPDATE conditions SET value = 4.00 WHERE time_int = 2;
--- This case updates values in the bucket, which affects two different partials in
--- two different chunks.
-UPDATE conditions SET value = 4.00 WHERE time_int = 9;
-UPDATE conditions SET value = 4.00 WHERE time_int = 11;
-SELECT timescaledb_experimental.refresh_continuous_aggregate('conditions_7', show_chunks('conditions', 20));
- refresh_continuous_aggregate 
-------------------------------
```

Comment:
All those chunk-versus-timebucket cases are not covered by other tests, so we shouldn't just remove them. Maybe figure out which ranges to use with the non-experimental `refresh_continuous_aggregate()`?

Reply:
I removed this test because removing the `chunk_id` from the materialization hypertable is not possible to it anymore, I mean refresh data from only one specific chunk. It will work just when we have the raw table data chunks aligned with the materialization hypertable chunks but it does not work for all cases.

Reply:
Can we rework those tests to work with ranges?

Reply:
Added some more tests!